database/index.js

```diff
@@ -13,7 +13,39 @@ connection.connect((err) => {
   console.log('Connected!');
 });
 
+const getDetails = (id, callback) => {
+  connection.query(`SELECT details FROM games where id="${id}"`, (err, results) => {
```

Comment:
this is good, but another way to write this is something like: `where id=?', id, (err...`  You pass the variable in by marking its place with a question mark and then use the variable in the second argument. You can actually pass in multiple variables this way with multiple question marks. And you don't use a template literal then, just regular quotes on the string. fwiw